src/background/adblocker.js

```diff
@@ -153,6 +158,12 @@ export const setup = asyncSetup('adblocker', [
   OptionsObserver.addListener(
     'experimentalFilters',
     async (value, lastValue) => {
+      if (__PLATFORM__ === 'firefox') {
+        EXPERIMENTAL_SCRIPTLETS_INJECTION = value;
```

Comment:
Please create another observer to separate logic. You are using the same option, but for another purpose.